src/fetch-wrapper.ts

```diff
@@ -27,6 +27,9 @@ export default class OAuth2 {
     if (!options.grantType && !token && !options.accessToken) {
       throw new Error('If no grantType is specified, a token must be provided');
     }
+    if (typeof options.scheduleRefresh === 'undefined') {
```

Comment:
This could be changed to 
```js
if (options.scheduleRefresh === undefined)
```

or simply:
```js
if (!options.scheduleRefresh)
```

Reply:
It needs to be a test on `undefined` because `false` would mean, don't schedule refresh, and the BC case (not specified) is to schedule refresh. Update incoming along with another PR.